nx-X11/programs/Xserver/include/resource.h

```diff
@@ -58,7 +58,6 @@ SOFTWARE.
 typedef unsigned long RESTYPE;
 
 #define RC_VANILLA	((RESTYPE)0)
-#define RC_CACHED	((RESTYPE)1<<31)
```

Comment:
I've been thinking about the reason for this and I think I understand why they don't delete the constant. Mostly for legacy-reasons. See, if old X11 client libraries set this, they expect a certain behavior. Deleting this constant from the server code may lead to someone re-introducing it for a different feature and then this will break badly with older X11 clients. So they just keep it around, doing nothing.